hr_timesheet_employee_analytic_tag/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+#. Go to *Timesheets* and creates a new record.
+#. Go to *Invoicing > Configuration > Management > Analytic Items*.
+#. The previously created record will have the employee's analytical tags defined.
```

Comment:
#. The previously created record will have the defined employee's analytical tags.